cluster/gce/gci/mounter/Dockerfile

```diff
@@ -16,5 +16,5 @@ FROM ubuntu:xenial
 MAINTAINER vishh@google.com
 
 RUN apt-get update && apt-get install -y netbase nfs-common=1:1.2.8-9ubuntu12 glusterfs-client=3.7.6-1ubuntu1
-
+ADD startmounter.sh /startmounter.sh
```

Comment:
You also need to update the `-exec` argument passed to `rkt` in the `mounter` script to point to this script.